cmd/neofs-cli/modules/object/hash.go

```diff
@@ -42,7 +42,7 @@ func initObjectHashCmd() {
 	flags.String(commonflags.OIDFlag, "", commonflags.OIDFlagUsage)
 	_ = objectHashCmd.MarkFlagRequired(commonflags.OIDFlag)
 
-	flags.String("range", "", "Range to take hash from in the form offset1:length1,...")
+	flags.String("range", "", "Range to take hash from in the form offset1:length1,... Full object payload length if not specified")
```

Comment:
can we print a verbose message in this case? (we have such practice in neofs-cli)